content/docs/hooks-state.md

```diff
@@ -212,69 +218,73 @@ Let's now **recap what we learned line by line** and check our understanding.
-* **Line 9:** When the user clicks, we call `setCount` with a new value. React will then re-render the `Example` component, passing the new `count` value to it.
+* **Satır 1:** React'dan `useState` Hook'u ekliyoruz. Bu, yerel state'i fonksiyon bileşeninde tutmamıza izin verecek.
+* **Satır 4:** `Example` bileşeninin içinde `useState` Hook'u çağırarak yeni bir state değişkeni tanımlıyoruz. Bu, isimlerini bizim verdiğimiz bir çift değer döndürür. Değişkenimizin adı `count` çünkü tıklama sayısını tutuyor.`useState`e `0` yollayarak `count`'u `0`sıfırdan başlatıyoruz. İkinci döndürülen değerin kendisi bir fonksiyondur. Bu, bize `count`'u güncellemek için izin verir. Bu yüzden onu `setCount` diye isimlendirdik.
+* **Satır 9:** Kullanıcı her tıkladığında, `setCount`'u yeni bir değerle çağırırız. React, `Example` bileşenini tekrar render edip, ona yeni `count` değerini atar.
+
+İlk bakışta öğrenecek çok fazla şey varmış gibi görünebilir. Acele etmeyin! Açıklamalar arasında kaybolduysanız, yukarıdaki koda tekrar bakın ve baştan sona tekrar okumaya çalışın. State'in sınıflarda nasıl çalıştığını unutup bu koda meraklı gözlerle tekrar baktığınızda, söz veriyoruz, daha anlamlı gelecek.
 
-This might seem like a lot to take in at first. Don't rush it! If you're lost in the explanation, look at the code above again and try to read it from top to bottom. We promise that once you try to "forget" how state works in classes, and look at this code with fresh eyes, it will make sense.
 
-### Tip: What Do Square Brackets Mean? {#tip-what-do-square-brackets-mean}
+### İpucu: Köşeli Parantez Ne Anlama gelir? {#tip-what-do-square-brackets-mean}
 
-You might have noticed the square brackets when we declare a state variable:
+State değişkenlerini tanımlarken köşeli parantezleri fark etmiş olabilirsiniz:
 
 ```js
   const [count, setCount] = useState(0);
 ```
 
-The names on the left aren't a part of the React API. You can name your own state variables:
+Soldaki isimler React API'inin bir parçası değildir.. State değişkeninizi isimlendirebilirsiniz:
 
 ```js
   const [fruit, setFruit] = useState('banana');
 ```
 
-This JavaScript syntax is called ["array destructuring"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring). It means that we're making two new variables `fruit` and `setFruit`, where `fruit` is set to the first value returned by `useState`, and `setFruit` is the second. It is equivalent to this code:
+Bu JavaScript sözdizimi ["array destructuring"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring) olarak adlandırılır.`fruit` ve `setFruit` diye iki yeni değişken oluşturduğumuz anlamına gelir.Burada `fruit` ve `useState` döndürür ve `setFruit` tarafından dönen ilk değere; `setFruit` ise ikinci değere atanır bu kod eşdeğerdir. 
```

Comment:
I think you skipped this part from my previous review. A possibly better translation

> Bu JavaScript sözdizimi `["dizi parçalama (array destructuring)"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring)` olarak adlandırılır. `fruit` ve `setFruit` diye iki yeni değişken oluşturduğumuz anlamina gelir. Burada `fruit`, `useState` tarafından dönen ilk değere; `setFruit` ise ikinci değere atanır. Bu kod ile eşdeğerdir:

Reply:
Add turkish translation :

> ["dizi parçalama (array destructuring)"]